index.css

```diff
@@ -0,0 +1,540 @@
+  .hamburger.active .bar:nth-child(1) {
+    transform: rotate(45deg) translateY(12px);
+  }
+
+  .hamburger.active .bar:nth-child(3) {
+    transform: rotate(-45deg) translateY(-11px);
+  }
+
+  .nav-menu {
+    position: fixed;
+    right: -100%;
+    height: 100vh;
+    z-index: 100;
+    top: -2px;
+    display: flex;
+    flex-direction: column;
+    gap: 2rem;
+    padding: 5rem 0;
+    align-items: center;
+    background-color: #ff5438;
+    mix-blend-mode: multiply;
+    backdrop-filter: blur(6px);
+    width: 100%;
+    transition: 0.5s;
+  }
+
+  .nav-menu.active {
+    right: 0;
+    z-index: 50;
+    overflow: hidden;
+  }
+
+  .nav-link {
+    font-family: 'Lato', sans-serif;
+    font-weight: 800;
+    font-size: 1.5rem;
+    color: #fff;
+    z-index: 9999;
+  }
+}
+
+.bar {
+  display: block;
+  width: 25px;
+  height: 5px;
+  border-radius: 1rem;
+  margin: 3px auto;
+  background-color: #595959;
+}
+
+.hero {
+  width: 100%;
+  background-image: url('./images/mobile-bg.png');
+  background-size: cover;
+  background-repeat: no-repeat;
+}
+
+.hero-wrapper {
+  width: 85%;
+  margin: 0 auto;
+  padding: 5rem 1rem 5rem 0;
+}
+
+.hello {
+  font-family: 'Lato', sans-serif;
+  color: #ff5438;
+  font-weight: 400;
+  font-size: 1rem;
+  line-height: 1rem;
+}
+
+.hero-wrapper h1 {
+  font-family: 'Cocogoose', sans-serif;
+  color: #ff5438;
+  font-weight: 700;
+  font-size: 1.8rem;
+  line-height: 2rem;
+  margin-bottom: 2rem;
+  background-clip: initial;
+  -webkit-background-clip: text;
+  -webkit-text-fill-color: transparent;
+  background-image:
+    -webkit-linear-gradient(
+      45deg,
+      #f40001,
+      #ec5242,
+      rgb(255, 64, 0),
+      rgb(255, 123, 0),
+      #a00000,
+      #f65112,
+      #ec5242
+    );
+}
+
+.description {
+  font-family: 'Lato', sans-serif;
+  color: #515151;
+  width: 89%;
+  background: #f8f8f8;
+  font-weight: 400;
+  font-size: 0.75rem;
+  line-height: 1rem;
+  border: 4px solid #fff;
+  padding: 0.5rem;
+  text-align: left;
+  margin-bottom: 2rem;
+}
+
+.date {
+  font-family: 'Gothic A1', sans-serif;
+  font-weight: 700;
+  font-size: 1.1rem;
+  color: #595959;
+  letter-spacing: -0.08em;
+  padding-bottom: 0.2rem;
+}
+
+.location {
+  font-family: 'Gothic A1', sans-serif;
+  font-weight: 600;
+  font-size: 0.7rem;
+  color: #868686;
+  letter-spacing: -0.08em;
+  padding-bottom: 0.2rem;
+}
+
+.program {
+  width: 100%;
+  background-image: url('./images/cross.svg');
+  background-repeat: repeat;
+  background-color: #282b32;
+  mix-blend-mode: multiply;
+}
+
+.program-wrapper {
+  width: 90%;
+  margin: 0 auto;
+  padding: 2rem 0;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.program-title {
+  font-family: 'Lato', sans-serif;
+  font-weight: 400;
+  font-size: 1.25rem;
+  line-height: 1.75rem;
+  color: #fff;
+  padding: 0.5rem;
+}
+
+.program-wrapper hr {
+  width: 15%;
+  border: 2px solid #a04038;
+  margin-bottom: 2rem;
+}
```

Comment:
- [ ] As suggested by the reviewer [here](https://github.com/ProgramKingAbel/Conference-site-/pull/1#discussion_r1180701352), please fix the line and it's color after **_Main Program_**. The color given in the design guidelines is `#ec5242`. Please fix this 🛠🤓


Your design | Design guidelines
---| ---| 
![image](https://user-images.githubusercontent.com/106119105/235274254-2a051bf3-aa57-4e33-8a0f-769f6f538530.png) | ![image](https://user-images.githubusercontent.com/106119105/235274285-4f88cf3c-0577-41a7-9683-2b0a89191ab1.png)